src/make_trade.ts

```diff
@@ -67,7 +68,7 @@ export async function makeTrade(
   );
 
   console.log(
-    `Selling ${sellAmountAfterFee.toString()} of ${
+    `🤹 Selling ${sellAmountAfterFee.toString()} of ${
```

Comment:
Wait, what?
You are not @anxolin, why are there emojis here?

Reply:
Love the emojis!